src/reporter.rs

```diff
@@ -24,7 +22,6 @@ impl ReportConfig {
         verbose: bool,
         strict: bool,
         ci_format: Option<CiFormat>,
-        output_file: Option<String>,
```

Comment:
Don't remove stuff. Or add in comment reason for removing.

Reply:
You've changed the positions. It changes the implementation and will need update everywhere.